test/e2e/autoscale_test.go

```diff
@@ -295,9 +313,9 @@ func assertNumberOfPods(ctx *testContext, numReplicasMin int32, numReplicasMax i
 		return errors.Wrapf(err, "Failed to get deployment %q", deployment)
 	}
 	gotReplicas := deployment.Status.Replicas
-	ctx.logger.Infof("Assert wanted replicas %d of deployment %s is between %d and %d replicas ", gotReplicas, ctx.deploymentName, numReplicasMin, numReplicasMax)
+	ctx.logger.Infof("Got %d replicas, expected between [%d, %d] replicas for deployment %s", gotReplicas, numReplicasMin, numReplicasMax, ctx.deploymentName)
```

Comment:
the exact string is repeated below. Perhaps mes := fmt.Sprintf(...) and then re-use?

Reply:
done